inspire_utils/record.py

```diff
@@ -80,3 +80,25 @@ def getitem(k, v, default):
         except KeyError:
             return default
     return value
+
+
+def get_values_for_schema(elements, schema):
+    """Return all values from elements having a given schema.
+
+    Args:
+        elements(List[dict]): a list of elements, all having at least the
+        ``schema`` and ``value`` fields.
+        schema(str): the schema that the values need to follow.
+
+    Returns:
+        list: all values conforming to the given schema.
+
+    Example:
+        >>> elements = [
+        ...     {'schema': 'TWITTER', 'value': 's_w_hawking'},
+        ...     {'schema': 'WIKIPEDIA', 'value': 'Stephen_Hawking'}
+        ... ]
+        >>> get_values_for_schema(elements, 'TWITTER')
+        ['Stephen_Hawking']
```

Comment:
Shouldn't it return ['s_w_hawking'] in this example?

Reply:
I fixed it